### PR TITLE
update hero links

### PR DESCRIPTION
**Pull request by luceliu**
Learn Day button links have been updated in Firebase

### CODE REVIEW

**Review by Ciuffi — APPROVED** [review]
Very nice

**Comment by Ciuffi** [comment]
@luceliu btw you have ot make sure to update the firebase production variables as well as the dev ones!

File: components/Intro.vue [hunk 1]
```diff
@@ -18,17 +18,16 @@
...
-        >
+        <p v-for="item in text" :key="item" class="date">
           {{ item }}
         </p>
         <Button title="Contact Us" url="mailto:lhd@nwplus.io" />
-        <Button disabled title="Sign Up" url="#" />
+        <Button
+          title="Sign Up"
+          url="https://organize.mlh.io/participants/events/1610-ubc-lhd-2019-learn-day"
+        />
```
Comment (acchiang) [inline components/Intro.vue:28]: any particular reason why the `<p>` tag was put into one line but the `<button>` was broken up into many lines? just curious

**Review by acchiang — APPROVED** [review]
looks good! thanks for covering all the bases (including the Learn Day button links in Firebase)!

**Pull request merged**